dotnet-engine/Yggdrasil.Engine/Types.cs

```diff
@@ -29,22 +29,20 @@ public class EngineResponse<TValue> : EngineResponse
 
 public class Variant
 {
-    public Variant(string name, Payload? payload, bool enabled, bool feature_enabled)
+    public Variant(string name, Payload? payload, bool enabled, bool featureEnabled)
     {
         Name = name;
         Payload = payload;
         Enabled = enabled;
-        Feature_Enabled = feature_enabled;
+        FeatureEnabled = featureEnabled;
     }
 
     public static readonly Variant DISABLED_VARIANT = new Variant("disabled", null, false, false);
 
     public string Name { get; set; }
     public Payload? Payload { get; set; }
-    [JsonPropertyName("enabled")]
     public bool Enabled { get; set; }
-    [JsonPropertyName("feature_enabled")]
```

Comment:
Shouldn't we keep this if we add it to the test case?

Reply:
Nah, wanna yet it. The point of this PR to stop dealing with "feature_enabled" and deal with "featureEnabled" instead so we can get consistent casing